pallets/authorship/src/tests.rs

```diff
@@ -0,0 +1,15 @@
+use frame_support::{assert_noop, assert_ok, BoundedVec};
+
+use cord_utilities::mock::mock_origin;
+use frame_system::RawOrigin;
+use sp_runtime::DispatchError;
+
+use crate::{mock::*, Banned, DidNameOwnershipOf, Error, Names, Owner, Pallet};
```

Comment:
review these

Reply:
Thanks. Got it. Will work on it and fix  issues asap.